### PR TITLE
[DX-1662] feat: add csharp core sdk docs

### PR DESCRIPTION
**Pull request by sprngbk**
## Description

This PR adds the C Sharp Core SDK docs to this repo

## Resolved issues
Jira: https://immutable.atlassian.net/browse/DX-1662

### Before submitting the PR, please consider the following:
- [ ] It's beneficial if your pull request references an issue used to discuss the pull request ahead of time. If you haven't previously created an issue, please create one and discuss your contribution with the maintainers.
- [ ] Prefix your PR title with `feat: `, `fix: `, `chore: `, `docs:`, or `refactor:`.
- [ ] The description should clearly illustrate what problems the pull request solves.
- [ ] Ensure that the commit messages follow our guidelines.
- [ ] Resolve merge conflicts (if any).
- [ ] Make sure that the current branch is up to date with the `main` branch.

### For internal Immutable developers:
- [ ] If you are adding or updating a documentation page, please also add or update the team ownership of that page (follow [this guide](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1947927045/Docs+ownership+registry#What-to-do-when-adding%2Fupdating-documentation-pages%3F)).
- [ ] If you are adding or updating documentation for an SDK, please make sure that you meet the requirements in [this doc](https://immutable.atlassian.net/wiki/spaces/PPS/pages/1916994017/SDK+documentation+guide).
- [ ] Please obtain PR approval from all of the following:
    - Your tech lead (Fallback: Team lead or engineering manager)
    - Your product manager (Fallback: Group PM)
    - Another software engineer on your team

### CODE REVIEW

File: docs/sdk-docs/core-sdk-csharp/additional-info.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+description: Additional info about the Golang Core SDK
```
Comment (suluvai) [inline docs/sdk-docs/core-sdk-csharp/additional-info.md:2]: Says, Golang

File: docs/sdk-docs/core-sdk-csharp/additional-info.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+id: additional-info
+slug: /additional-info
+tags: [core-sdk-golang]
```
Comment (suluvai) [inline docs/sdk-docs/core-sdk-csharp/additional-info.md:5]: Golang?

File: src/constants.tsx [hunk 3]
```diff
@@ -89,7 +96,8 @@ export const sdkIds = {
   coreKotlin: coreSDKs.find((sdk) => sdk.id === 1).sdkId,
   coreSwift: coreSDKs.find((sdk) => sdk.id === 2).sdkId,
   coreGolang: coreSDKs.find((sdk) => sdk.id === 3).sdkId,
-  walletWeb: walletSDKs.find((sdk) => sdk.id === 4).sdkId,
-  walletAndroid: walletSDKs.find((sdk) => sdk.id === 5).sdkId,
-  walletIOS: walletSDKs.find((sdk) => sdk.id === 6).sdkId,
+  coreCSharp: coreSDKs.find((sdk) => sdk.id === 4).sdkId,
+  walletWeb: walletSDKs.find((sdk) => sdk.id === 5).sdkId,
+  walletAndroid: walletSDKs.find((sdk) => sdk.id === 6).sdkId,
+  walletIOS: walletSDKs.find((sdk) => sdk.id === 7).sdkId,
```
Comment (carleeto) [inline src/constants.tsx:102]: The IDs of the web, android and iOS sdks will change as a result - is this ok or should the C# sdk have an ID of 7?

Comment (carleeto) [inline src/constants.tsx:102]: The IDs of the web, android and iOS sdks will change as a result - is this ok or should the C# sdk have an ID of 7?

**Comment by suluvai** [comment]
No csharp sdk?

Screenshot 2023-01-27 at 9 02 32 am

File: src/constants.tsx (hunk 3 above)
Comment (sprngbk) [reply src/constants.tsx:102]: Yup that's OK as long as we update the references, and this is done

**Comment by sprngbk** [comment]
> https://deploy-preview-215--imx-docs-prod.netlify.app

It's there:
image

**Comment by suluvai** [comment]
May need to add the package reference link once the package is published to Nuget.

**Review by suluvai — APPROVED** [review]

**Pull request merged**